src/themes/cerulean/globals/site.variables

```diff
@@ -19,3 +19,5 @@
 @warningBorderColor: darken(spin(@warningBackgroundColor, -10), 3%);
 @negativeTextColor: #b94a48;
 @negativeBorderColor: darken(spin(@negativeBackgroundColor, -10), 3%);
+
+@fontName: 'Helvetica Neue';
```

Comment:
Default `site.variables`:  https://github.com/semantic-ui-forest/semantic-ui/blob/master/src/themes/default/globals/site.variables#L9-L21

```
@fontName          : 'Lato';
@fontSmoothing     : antialiased;

@headerFont        : @fontName, 'Helvetica Neue', Arial, Helvetica, sans-serif;
@pageFont          : @fontName, 'Helvetica Neue', Arial, Helvetica, sans-serif;

@googleFontName    : @fontName;
@importGoogleFonts : true;
@googleFontSizes   : '400,700,400italic,700italic';
@googleSubset      : 'latin';

@googleProtocol    : 'https://';
@googleFontRequest : '@{googleFontName}:@{googleFontSizes}&subset=@{googleSubset}';
```

I think if one theme does not use Google font, then we should set `@importGoogleFonts : true;` to `@importGoogleFonts : false;`.

And, besides set `@fontName: 'Helvetica Neue'`, it's better to modify `@headerFont` and `@pageFont` to remove 'Helvetica Neue' duplicate.